tests/test_servefile.py

```diff
@@ -370,7 +388,7 @@ def test_tar_compression(run_servefile, datadir):
     p = datadir(d)
     run_servefile(['-c', 'gzip', '-t', str(p / 'foo')])
 
-    r = make_request()
+    r = _retry_while(requests.exceptions.ConnectionError, make_request)()
```

Comment:
you're using the long form of the exception until here, and below it's only `ConnectionError` which you import below. maybe just remove the `import` and always use the long form?

Reply:
That's an unintended consistency, yes. You don't seem to like from imports really :smile: I'll use the short version everywhere now, happy to adjust for @sebageek to long form as needed.

Reply:
I like the short form better here, especially as it is used within so many calls :)